README.md

```diff
@@ -1,32 +1,44 @@
 # Share Button Study
-The purpose of this extension is to visually highlight the share button in the browser toolbar when text from the URL bar is copied.
+![CircleCI badge](https://img.shields.io/circleci/project/github/marcrowo/share-button-study/master.svg?label=CircleCI)
 
-## Overview of extension
-See [Bootstrapped extensions](https://developer.mozilla.org/en-US/Add-ons/Bootstrapped_extensions) for more information.
+The purpose of this extension is to visually highlight the share button in the browser toolbar when text from the URL bar is copied, and to serve as a template for new Shield studies.
 
-### bootstrap.js
-MDN References [[1](https://developer.mozilla.org/en-US/docs/Extensions/bootstrap.js)] [[2](https://developer.mozilla.org/en-US/Add-ons/Bootstrapped_extensions#Bootstrap_entry_points)]
+## Overview
+The share button study is a bootstrapped extension. See [Bootstrapped extensions](https://developer.mozilla.org/en-US/Add-ons/Bootstrapped_extensions) for more information.
 
-`startup()` 
-1. Load the stylesheet for the share button animation.
-2. Add the controller that detects the copy event in the URL bar.
-3. Add an eventListener to the share button so that the CSS animation can be replayed.
+As a starting point, you may want to read through `extension/bootstrap.js`.
 
-`shutdown()`
-1. Remove the CSS.
-2. Remove the animation class from the share button.
-3. Remove the controller from the URL bar.
-4. Remove the eventListener from the share button.
+## Setup
 
-### chrome.manifest
-[MDN Reference](https://developer.mozilla.org/en-US/docs/Chrome_Registration)
+### Requirements
+- [yarn](https://yarnpkg.com)
```

Comment:
You don't need yarn to run the experiment; you can run it in `npm` if you remove all the *carets* from the dependencies. Not sure which dependencies are the ones causing issue.

Reply:
Removing all the carets from the dependencies will only affect the top-level dependencies.
This does not work for a dependency of a dependency, ie.:

package.json includes PACKAGE_A version 1.0
PACKAGE_A requires PACKAGE_B.

Removing carets will from package.json will ensure that PACKAGE_A will be the right version, but does not guarantee that its dependency, namely PACKAGE_B, will be the right version.

In this case, the dependency issue happened to be a top-level dependency and therefore the caret trick will work, but this will not always be the case.